src/web/jcc/client/components/RCControllerPanel.tsx

```diff
@@ -86,30 +90,65 @@ export default class RCControllerPanel extends React.Component {
             // clicks on a new bot details window
             botId: 0,
             isMaximized: true,
+            overdriveEnabled: false,
         };
     }
 
     updateThrottleDirectionMove(event: IJoystickUpdateEvent) {
```

Comment:
We should add a comment for this function since it was changed in this pull request.